kube/pod_test.go

```diff
@@ -198,18 +222,19 @@ func TestPodGetEnvVars(t *testing.T) {
 		founda := false
 		foundb := false
 		foundc := false
-		for _, result := range vars {
-			if result.Name == "SOME_VAR" {
+		for _, result := range vars.Values() {
+			name := result.Get("name").Value()
+			if name == "SOME_VAR" {
```

Comment:
I added the `switch` but otherwise left it as-is. The test is kind of odd, as it executes several identical checks, but it was already like this before, and with this change we are leaving it at least somewhat better than we found it. 😄